relay/sources/relayd/src/data/shared_file.rs

```diff
@@ -57,7 +57,7 @@ impl SharedFile {
         // Only ascii alphanumeric, - and .
         // This is the documented constraint for file_id
         // More than enough for node ids too but we don't have a precise spec
-        let check = Regex::new(r"^[(?-u:\w)\-.]+$").unwrap();
+        let check = Regex::new(r"^[A-Za-z0-9\-.]+$").unwrap();
```

Comment:
this prevented unicode but still allowed extended ascii. No need for this, let's restrict more.